native/packages/react-native-bpk-component-card/readme.md

```diff
@@ -0,0 +1,57 @@
+# react-native-bpk-component-card
+
+> Backpack React Native card component.
+
+## Installation
+
+```sh
+npm install react-native-bpk-component-card --save-dev
+```
+
+## Usage
+
+```js
+import React, { Component } from 'react';
+import { View, StyleSheet, Text } from 'react-native';
+import BpkCard from 'react-native-bpk-component-card';
+import BpkText from 'react-native-bpk-component-text';
+import * as TOKENS from 'bpk-tokens/tokens/ios/base.react.native.es6';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    justifyContent: 'center',
+    padding: TOKENS.spacingBase,
+  }
+});
+
+export default class App extends Component {
+  render() {
+    const copy = (
```

Comment:
Very small thing but I'm wondering if something like `cardContent` would be more explicit as a variable name?